test/compflags/bradc/minimalModules/SKIPIF

```diff
@@ -4,3 +4,4 @@ CHPL_COMM != none
 COMPOPTS <= --no-local
 CHPL_LOCALE_MODEL != flat
 EXECOPTS <= memLeaks
+CHPL_LLVM == none
```

Comment:
If we document `--minimal-modules` anywhere, I think you should note this limitation there. If we don't, then no need and ignore this comment

Reply:
I am not aware of such docs.